packages/plugin-expo-app/test/app.test.js

```diff
@@ -3,38 +3,7 @@
 const proxyquire = require('proxyquire').noPreserveCache().noCallThru()
 const Client = require('@bugsnag/core/client')
 
-describe('plugin: expo device', () => {
-  it('should should use version if defined (all platforms)', done => {
-    const VERSION = '1.0.0'
-    const plugin = proxyquire('../', {
-      'expo-constants': {
-        default: {
-          platform: {
-          },
-          manifest: { version: VERSION }
```

Comment:
I don't think there's any test coverage ensuring we capture this now

Reply:
It's not covered by unit tests anymore but it is covered by this maze-runner step:

https://github.com/bugsnag/bugsnag-js/blob/9fcf7efbccd2cad74079ae89256c090c6c6ea640/test/expo/features/app.feature#L12
which asserts that the value is passed through from the fixture app's `app.json`:
https://github.com/bugsnag/bugsnag-js/blob/9fcf7efbccd2cad74079ae89256c090c6c6ea640/test/expo/features/fixtures/test-app/app.json#L11